var/spack/repos/builtin/packages/intel-oneapi-mkl/package.py

```diff
@@ -31,3 +31,30 @@ def __init__(self, spec):
                             releases=releases,
                             url_name='onemkl')
         super(IntelOneapiMkl, self).__init__(spec)
+    #FTW: Need to work in paths for  /soft/spack/opt/spack/linux-opensuse_leap15-cascadelake/oneapi-2021.1/intel-oneapi-mkl-2021.1.1-n2zoxk4l5w62slfcyp3sdirrrxp77bko/mkl/latest/../2021.1.1/lib/intel64 (all the mkl stuff) plus tbb: /soft/spack/opt/spack/linux-opensuse_leap15-cascadelake/gcc-10.1.0/intel-oneapi-compilers-2021.1.0-wgzjfktx5hsx75gghtd6k2qxad5vnot2/tbb/2021.1.1/lib/intel64/gcc4.8
+    def _join_prefix(self, path):
+        return join_path(self.prefix, 'mkl', 'latest', path)
+
+    def _ld_library_path(self):
+        dirs = ['lib/intel64']
+        for dir in dirs:
+            yield self._join_prefix(dir)
+
+    def _library_path(self):
+        dirs = ['lib/intel64']
+        for dir in dirs:
+            yield self._join_prefix(dir)
+
+    def install(self, spec, prefix):
```

Comment:
I think this will be inherited if you delete it. Or did you add it because there was a problem?

Reply:
It was left over from some cut and paste. I've deleted it.